.pylintrc

```diff
@@ -82,3 +82,8 @@ disable=
 	# if:/else: and for:/else:.
 	useless-else-on-loop,
 	no-else-return,
+	useless-object-inheritance,
```

Comment:
Are you certain that you want to globally disable these warnings? If there are specific issues, you should favor refactoring and/or suppressing warnings at those specific code points, not a global suppression.

Reply:
For disabled rules, I either added description or open an issue for them.